app/console.go

```diff
@@ -35,30 +37,36 @@ func RunConsole(ps *peer.PeerStore) {
 	shell.AddCmd(&ishell.Cmd{
 		Name: "address",
 		Help: "show the address this host is listening on",
-		Func: listenAddr,
+		Func: func(ctx *ishell.Context) {
+			listenAddr(ctx, peerStore)
+		},
 	})
 	shell.AddCmd(&ishell.Cmd{
 		Name: "peers",
 		Help: "show the peers this host is connected to",
-		Func: peers,
+		Func: func(ctx *ishell.Context) {
```

Comment:
I see now that you're doing it because you only have access to `app` from `Run()`, but that need not be the case in my opinion.